### PR TITLE
Update first days notebooks

### PR DESCRIPTION
**Pull request by mcollardanuy**

### CODE REVIEW

File: Sessions/1b-Intro-to-Python.ipynb [hunk 1]
```diff
@@ -1,35 +1,48 @@
...
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
    "outputs": [],
    "source": [
     "text = \"This is our first python example.\"\n",
```
Comment (kasparvonbeelen) [inline Sessions/1b-Intro-to-Python.ipynb]: Python with a capital :)?

File: Sessions/2-opening-files.ipynb [hunk 2]
```diff
@@ -22,143 +30,241 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "fbfc7e7a",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
     "* **Relative path:** The location of a file or folder relative to the location of the python script or notebook. For example, from this notebook, the relative path to the file `tale_beginning.txt` is:\n",
```
Comment (kasparvonbeelen) [inline Sessions/2-opening-files.ipynb]: maybe mention 'relative to pwd`?

**Review by kasparvonbeelen — DISMISSED** [review]
Besides two minor comments, this looks good to me!

File: Sessions/1b-Intro-to-Python.ipynb (hunk 1 above)
Comment (mcollardanuy) [reply Sessions/1b-Intro-to-Python.ipynb]: Done!

File: Sessions/2-opening-files.ipynb (hunk 2 above)
Comment (mcollardanuy) [reply Sessions/2-opening-files.ipynb]: Done!

**Comment by mcollardanuy** [comment]
Thanks @kasparvonbeelen! I think you need to approve the changes again!

**Review by kasparvonbeelen — APPROVED** [review]
LGTM!

**Pull request merged**